src/pages/home/styled/ProfileImage.ts

```diff
@@ -0,0 +1,11 @@
+import { styled } from "@mui/material/styles";
+
+const ProfileImage = styled("img")(
+  ({ theme }) => `
+  border-radius: 50%;
+  width: ${theme.spacing(16.25)};
+  height: ${theme.spacing(16.25)};
+`
+);
+
+export default ProfileImage;
```

Comment:
I was holding off on adding `styled` and `component` directories inside of the `pages` directory, since it can be confusing to have components in multiple directories, but I think it does make sense to have the components here if they are only used on a single page